en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,274 @@
+# OMP Hobby ZMO FPV
+
+The OMP Hobby ZMO is a small sized tiltrotor VTOL that is available as RTF kit.
+
+![Finished ZMO hover 1](../../assets/airframes/vtol/omp_hobby_zmo_fpv/airframe-hover.jpg)
+
+![Finished ZMO hover 2](../../assets/airframes/vtol/omp_hobby_zmo_fpv/airframe-hover2.jpg)
+
+
+## Overview
+
+Key airframe features:
+
+- Spacious interior
+- Pre installed actuators
+- Quick release wing connecting system
+- ~35 minute long flight times (depending on takeoff weight)
+- VTOL enables flying in locations where a fixed wing couldn't fly
+- Battery and charger included in the kit
+- Easy overall build
+- Space to mount FPV and/or action camera in the front
+- Airspeed data from pitot/static probe
+- Lidar distance sensor
+
+Depending on the final takeoff weight the hover time might be limited, since there is not a lot of air circulation inside the fuselage when the vehicle is hovering.
+
+## Where to buy
+
+- [OMP-Hobby](https://www.omphobby.com/OMPHOBBY-ZMO-VTOL-FPV-Aircraft-With-DJI-Goggles-And-Remote-Controller-p3069854.html)
+- [GetFPV](https://www.getfpv.com/omphobby-zmo-z3-vtol-fpv-1200mm-arf-plane-kit-no-fpv-system.html)
+- [FoxtechFPV](https://www.foxtechfpv.com/zmo-pro-fpv-vtol.html)
+
+## Flight controller options

```

Comment:
Note, there are many many more options than this. SO let's be more clear and state that these are options that have been tried.

```suggestion
## Flight Controller

The following options have been tested:
```